pkgs/development/python-modules/face_recognition/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildPythonPackage, fetchPypi, pillow, click, dlib, numpy
+, face_recognition_models, scipy, stdenv
+}:
+
+buildPythonPackage rec {
+  pname = "face_recognition";
+  version = "1.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0h2bjbkh6zqn7053yjyzlqvdq7ivjjjjdnpzbczi25dmhc74v4dw";
+  };
+
+  propagatedBuildInputs = [ pillow click dlib numpy face_recognition_models scipy ];
+
+  doCheck = false;
```

Comment:
Reason?

Reply:
my bad, they're not available with `pypi`, moving to `fetchFromGitHub` + active tests, just did some fast bootstrapping in an isolated environment, but you're right, active tests are way better %)